src/routes/blog/ide-ux-bof-scipy-2024/+page.md

```diff
@@ -0,0 +1,141 @@
+Another had concerns with the usability of Zoom screen sharing, as sharing a desktop application just shared that application, while sharing a web-based IDE shared the user's entire web browser.
+A third suggested [vscode.dev](https://vscode.dev/), which runs in a browser and which can be self-hosted, but stated that Python support is not "fully on par" with the desktop version.
+A final point discussed was the possibility of running desktop IDEs inside a web-based container to improve security and ease of deployment, particularly using Python's new WASM support.
+
+On a related note, an attendee shared that they support a small statics group, and their biggest challenge is helping students learn scientific computing.
+They have a rather "rugged individualist culture", and feel they need to learn how to share knowledge with others, asking the audience how they could accomplish that with IDEs.
+This prompted others to related how that where they work, platforms for accessing data require a lot of authentication, and entail using JupyterHub as their default IDE/platform, with very little flexibility in the matter.
+They would like a platform for other IDEs to be able to connect to a shared JupyterHub environment, which confidential is exactly what Spyder is working on for Spyder 6 and beyond.
+
+
+## GPU compute integration
+
+One fairly specific topic that got a disproportionate amount of attention was incorporating better support for GPU compute (e.g. CUDA) within IDEs and interfaces, thanks to Nvidia representatives in the room posing the initial question.
+They stated that Nvidia was "getting serious" about Python and CUDA, and they would love to "hear people's gripes about developing GPU apps in Python, what tooling you use and what problems you face and how do you debug them".
+They also asked for people's experiences working in Python together with native code (C++).
+
+Responding first to the question on native code, one participant described how they had to compile a Fortran module and import it from Python.
+They just used `f2py` and the workflow was already in there and worked: they'd change the native code, recompile and rinse and repeat.
+However, they said it Would be nice to offer a workflow to shorten that development loop.
+It was mentioned that this was particularly a problem with two separate codebases, where changes in one can affect the other, which is difficult to conceptualize and work with the coupling between them.
+The Nvidia representatives then asked them how they would like to have a debugger with the ability to step into their code from Python to Fortran but still able to inspect your variables on the Python side, to which they replied "Sounds awesome!".
+
+Another attendee has a postdoc using a bunch of A100 GPUs, and asked them whether they were getting the most performance out of it.
```

Comment:
So this was a PI asking their own postdoc if they were getting the most perfomance out of their GPUs? If so, I think this sentence could be rephrased to make it clearer because it's a bit confusing as it is (I don't have a suggestion in mind).